.gitignore

```diff
@@ -87,6 +87,9 @@ ipch/
 *.sdf
 *.cachefile
 
+# Visual Studio cache directory
+.vs
```

Comment:
I have also noticed this and this new ignore should be applied in all of our repos, thanks for confirming